### ISSUE TITLE
Review H8_PTP_Final_Image

### ISSUE THREAD

**Issue by syahrulhamdani** [IndraPratamaLin/H8_PTP_Final_Image#1]
* You may want to add some reasoning why you used a grayscale image as your training data in the notebook (?)
* Some remarks on why the validation loss is lower than training loss: it may be because your model is doing a pretty good job. Thus, you may increase the epochs and continue to train the model.
* You may want to add some image visualizations in your notebook, both right after preprocessing your input data and after predicting test data.

Overall, good job!
